facturacion_mexico/setup/enforce_sat_uom.py

```
@@ -0,0 +1,64 @@
+		enabled = bool(uom["enabled"])
+
+		if name.startswith("_Test"):
+			continue
+
+		if name in SAT_UOMS:
+			if not enabled:
+				frappe.db.set_value("UOM", name, "enabled", 1, update_modified=False)
+				sat_corregidas += 1
+			sat_verificadas += 1
+		else:
+			if enabled:
```

Ruff (0.15.14)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)